### ISSUE TITLE
Encoding Problem when calculating the Lenght of the Response contentLenght

### ISSUE THREAD

**Issue by sijakubo** [graphql-java-kickstart/graphql-java-servlet#223]
Hi, 

the servlet responses are cut off when containing utf8 charachters like 'ä', 'ö', ...

I've noticed that you already fixed this issue in your current master. Could you release a new Version with this fix? This is a show stopper for us.

```
response.setContentLength(responseContent.getBytes(StandardCharsets.UTF_8).length);
```

Best regards,
Simon Jakuobwski

**Comment by oliemansm** [comment]
It’s already released in 9.0.1. Just didn’t have time yet for release notes

Groeten,
Michiel
________________________________
Van: Simon Jakubowski <notifications@github.com>
Verzonden: Friday, December 13, 2019 3:10:59 PM
Aan: graphql-java-kickstart/graphql-java-servlet <graphql-java-servlet@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Onderwerp: [graphql-java-kickstart/graphql-java-servlet] Encoding Problem when calculating the Lenght of the Response contentLenght (#223)


Hi,

the servlet responses are cut off when containing utf8 charachters like 'ä', 'ö', ...

I've noticed that you already fixed this issue in your current master. Could you release a new Version with this fix? This is a show stopper for us.

Best regards,
Simon Jakuobwski

**Comment by sijakubo** [comment]
9.0.1 fixed my issue. Thank you
